drivers/kscan/Kconfig.cst816s

```
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 Qingsong Gou <gouqs@hotmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_HYNITRON_CST816S := hynitron,cst816s
+
+menuconfig KSCAN_CST816S
+	bool "CST816S capacitive touch panel driver"
+	default $(dt_compat_enabled,$(DT_COMPAT_HYNITRON_CST816S))
+	depends on I2C
+	help
+	  Enable driver for hynitron cst816s touch panel.
+
+if KSCAN_CST816S
+
+config KSCAN_CST816S_PERIOD
+	int "Sample period"
+	depends on !KSCAN_CST816S_INTERRUPT
+	default 20
+	help
+	  Sample period in milliseconds when in polling mode.
+
+config KSCAN_CST816S_INTERRUPT
+	bool "Enable interrupt support"
+	default $(dt_compat_enabled,$(DT_COMPAT_HYNITRON_CST816S))
```

this seems redundant, please change to:

```suggestion
	default y
```